core/sql/optimizer/RelRoutine.cpp

```diff
@@ -1027,7 +1052,7 @@ OperatorTypeEnum PredefinedTableMappingFunction::nameIsAPredefinedTMF(const Corr
   const NAString &funcName = qualName.getObjectName();
```

Comment:
Maybe small, but still a valid point! I stumbled over this while re-reading the code. Fixed.